sw/device/silicon_creator/manuf/lib/personalize_functest.c

```diff
@@ -65,6 +65,21 @@ bool test_main(void) {
   ujson_t uj = ujson_ottf_console();
   CHECK_STATUS_OK(peripheral_handles_init());
 
+  // If we are in the RMA state, this means that the personalization is complete
+  // and that the host has issued an LC transition to the RMA state. To allow
+  // the host to connect over JTAG and check that we have indeed reached this
+  // state, just spin to prevent a reboot.
+  dif_lc_ctrl_state_t lc_state = kDifLcCtrlStateInvalid;
+  CHECK_DIF_OK(dif_lc_ctrl_get_state(&lc_ctrl, &lc_state));
+  if (lc_state == kDifLcCtrlStateRma) {
+    LOG_INFO("Now in RMA state, spinning for host to connect over JTAG.");
+    // Wait in a loop so that OpenOCD can connect to the TAP without the ROM
+    // resetting the chip.
+    while (1) {
+      // Do nothing
```

Comment:
Yes that's a good point, but I don't like it because unless you read the code it's not clear that abort just spins. I am happy to add a `wait_for_interrupt` in the loops or create a small function `spin_for_jtag` in the file.